polish_labels.json

```diff
@@ -0,0 +1,62 @@
+{
+    "alarm_query": "alarm_zapytanie",
+    "alarm_remove": "alarm_usuwanie",
+    "alarm_set": "alarm_ustawianie",
+    "audio_volume_down": "dźwięk_ciszej",
+    "audio_volume_mute": "dźwięk_wycisz",
+    "audio_volume_other": "dźwięk_inne",
+    "audio_volume_up": "dźwięk_głośniej",
+    "calendar_query": "kalendarz_zapytanie",
+    "calendar_remove": "kalendarz_usuwanie",
+    "calendar_set": "kalendarz_ustawianie",
+    "cooking_query": "gotowanie_zapytanie",
+    "cooking_recipe": "gotowanie_przepis",
+    "datetime_convert": "data_czas_konwersja",
+    "datetime_query": "data_czas_zapytanie",
+    "email_addcontact": "email_dodawanie_kontaktu",
+    "email_query": "email_zapytanie",
+    "email_querycontact": "email_zapytanie_kontakt",
+    "email_sendemail": "email_wysyłanie",
+    "general_greet": "ogólne_powitanie",
+    "general_joke": "ogólne_żart",
+    "general_quirky": "ogólne_zapytanie",
```

Comment:
`'query'` seems to fit here better than `'quirky'` - that's way I use `'zapytanie'` here